### DISCUSSION TITLE
What's the keymap for searching text in a window?

### DISCUSSION THREAD

**Discussion by andradei** [kovidgoyal/kitty#8045] [Q&A] [answered]
I tried searching usually it's Ctrl-Shift-f or Ctrl-Shift-s
But I haven't found this basic feature in the docs.

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/overview/#the-scrollback-buffer

**Comment by andradei** [comment]
This feature is powerful, though unconventional. Thank you for the answer and for this impressive piece of software.
